api/taskgroup.go

```diff
@@ -136,20 +136,27 @@ func (h TaskGroupHandler) Update(ctx *gin.Context) {
 	if err != nil {
 		return
 	}
-	g := r.Model()
-	g.ID = id
+	g := &model.TaskGroup{}
+	result := h.DB.First(g, id)
+	if result.Error != nil {
+		h.getFailed(ctx, result.Error)
+		return
+	}
+	updated := r.Model()
+	updated.ID = g.ID
+	updated.Bucket = g.Bucket
```

Comment:
Mainly need to fetch the group to ensure the bucket is propagated.  The caller should PUT a complete object fetched with GET but no guarantees and the bucket is too important.